rllib/core/rl_module/marl_module.py

```diff
@@ -0,0 +1,295 @@
+from typing import Iterator, Mapping, Any, Union, Dict, Set
+import pprint
+
+
+from ray.rllib.utils.annotations import (
+    OverrideToImplementCustomLogic,
+    override,
+)
+
+from ray.rllib.models.specs.specs_dict import ModelSpec
+from ray.rllib.policy.sample_batch import MultiAgentBatch
+from ray.rllib.core.rl_module import RLModule
+
+ModuleID = str
+
+
+class MultiAgentRLModule(RLModule):
+    """Base class for multi-agent RLModules.
+
+    This class holds a mapping from module_ids to RLModules. It provides a convenient
+    way of accessing each individual module as well as all of with only one call.
+
+    The default implementation assumes the data communicated as input and output of
+    RLModule APIs are multi-agent batches. Each RLModule simply takes its own input and
```

Comment:
input and output of **RLModule APIs** or **MultiAgentRLModule APIs** ?

In other words. Does RLModule already define this constraint? If yes, we should not talk so much about RLModule here, but the specifics of MultiAgentRLModule as a subclass of it.

Reply:
sorry for the confusion, I have clarified the docstring.